chainstate/transaction_test.go

```diff
@@ -161,6 +162,7 @@ func TestClient_Transaction_BroadcastClient(t *testing.T) {
 			Build()
 		c := NewTestClient(
 			context.Background(), t,
+			WithMinercraft(&MinerCraftBase{}),
```

Comment:
This should not be here, because it's a test for go-broadcast-client.

Reply:
Unfortunately, this is a dependency necessary to run the client :(

Reply:
Here is a thing. We always need some MinecraftClient on start of Bux, as we need to prepare fee quotes

Reply:
To prepare fee quotes and validate miners. And as Arek said, without minercraft client starting of bux will fail

Reply:
Okay, I understand